contacts.module

```diff
@@ -75,17 +66,15 @@ function template_preprocess_contacts_dash_tabs(array &$variables) {
 }
 
 /**
- * Prepares variables for contact tab content.
+ * Prepares variables for contact tab content stacked layout.
  *
- * Default template: contact-tab-content.html.twig.
+ * Default template: contacts-tab-content-stacked.html.twig.
  *
- * @param array $variables
- *   An associative array containing:
- *   - attributes: HTML markup attributes for the content wrapper.
- *   - region_attributes: HTML markup attributes for the content wrapper.
+ * @see template_preprocess_contacts_dash_tab_content()
  */
-function template_preprocess_contact_tab_content(array &$variables) {
-  $variables['region_attributes'] = new Attribute($variables['region_attributes']);
+function template_preprocess_contacts_tab_content_stacked(array &$variables) {
```

Comment:
Let's drop the pre-process, we can use `$content['region']['#prefix']` etc to add additional wrappers as required for manage mode.